egs/multi_conv_zh_es_ta/ST/prepare.sh

```
@@ -0,0 +1,175 @@
+    fi
+    ./local/train_bpe_model.py \
+      --lang-dir $lang_dir \
+      --vocab-size $vocab_size \
+      --transcript $lang_dir/transcript_words.txt
+  done
+fi
+
+
+if [ $stage -le 7 ] && [ $stop_stage -ge 7 ]; then
+  log "Stage 7: Prepare BPE ST based lang"
+  for vocab_size in ${st_vocab_sizes[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)